rust/agama-lib/src/files/store.rs

```diff
@@ -55,7 +55,25 @@ impl FilesStore {
     }
 
     /// stores the list of user files via http API
-    pub async fn store(&self, files: &Vec<UserFile>) -> FilesStoreResult<()> {
-        Ok(self.files_client.set_files(files).await?)
+    pub async fn store(
+        &self,
+        files: &Vec<UserFile>,
+        context: &StoreContext,
+    ) -> FilesStoreResult<()> {
+        let resolved_files = if let Some(base) = &context.source {
+            &files
+                .iter()
+                .filter_map(|file| {
+                    file.resolve_url(&base)
+                        .inspect_err(|e| {
+                            log::warn!("Error processing file {}: {e}", file.destination)
```

Comment:
I don't understand why we only log the error (and omit the erroneous Files from the store). Shouldn't the user see the failure?